src/hubspace/hubspace_user.py

```diff
@@ -7,8 +7,8 @@
 import datetime
 import json
 
-_default_duration = 120 * 1000
-_default_expiration_buffer = 10 * 1000
+_default_duration = 120 #seconds
+_default_expiration_buffer = 10 #seconds
```

Comment:
the time is defined in seconds not milliseconds for this API (at least right now it's defined in seconds)